<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph"/>
   </widget>
   <widget class="QPushButton" name="pid_input">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>입력완료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>571</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_textClear">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>항목삭제</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>에스코어 드림 9 Black</family>
     </font>
    </property>
    <property name="title">
     <string>설정값</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>에스코어 드림 7 ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kp</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>에스코어 드림 7 ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ki</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>에스코어 드림 7 ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kd</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="kp_input"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ki_input"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="kd_input"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>에스코어 드림 7 ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>목표값</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sp_input"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>에스코어 드림 9 Black</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>제어타입</string>
    </property>
    <widget class="QComboBox" name="cmb_contType">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>208</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>에스코어 드림 1 Thin</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>** Control Type을 선택하세요 **</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1. Velocity Control Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2. Position Control Type</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>에스코어 드림 9 Black</family>
     </font>
    </property>
    <property name="title">
     <string>제어기 출력 선택</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_type1">
        <property name="font">
         <font>
          <family>에스코어 드림 9 Black</family>
         </font>
        </property>
        <property name="text">
         <string>Type1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_type2">
        <property name="font">
         <font>
          <family>에스코어 드림 9 Black</family>
         </font>
        </property>
        <property name="text">
         <string>Type2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_type3">
        <property name="font">
         <font>
          <family>에스코어 드림 9 Black</family>
         </font>
        </property>
        <property name="text">
         <string>Type3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
